Gemfile

```diff
@@ -4,33 +4,41 @@ gemspec
 
 group :debug do
   gem "pry"
-  gem "pry-byebug"
-  gem "pry-stack_explorer", "= 0.4.11" # remove this pin when we drop support for Ruby < 2.6
```

Comment:
@jeremiahsnapp we're not using this dep so we can just nuke it instead of bothering with a pin